src/content/ui-api/kubectl-gs/login.md

```diff
@@ -31,18 +31,18 @@ Note that `2` implies `1`. When setting up workload cluster access, management c
 The command is called with the general syntax
 
 ```nohighlight
-kubectl gs login <management-cluster> [flags]
+kubectl gs login [MANAGEMENT_CLUSTER] [FLAGS]
```

Comment:
Just for my understanding, when do we use the square brackets? I'd assumed it was to indicate the placeholder represents a variable number of items, but we use it here with `[MANAGEMENT_CLUSTER]` as well

Reply:
Square brackets are conventional for optional arguments. It's in fact possible to call `kubectl gs login` just like that.